### ISSUE TITLE
After some editing with new editor (upload images, links and alignment) mobilization render crash

### ISSUE THREAD

**Issue by lpirola** [nossas/bonde#253]
Error: Minified exception occurred; use the non-minified dev environment for the full error message and

https://sentry.io/nossas-cidades/usuario-final/issues/175693447/

```
Error: Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings.
  at invariant (~/react/~/fbjs/lib/invariant.js:35)
  at Object.findComponentRoot (~/react/lib/ReactMount.js:764)
  at Object.findReactNodeByID (~/react/lib/ReactMount.js:679)
  at Object.getNode (~/react/lib/ReactMount.js:162)
  at Object.updatePropertyByID (~/react/lib/ReactDOMIDOperations.js:48)
...
(45 additional frame(s) were not displayed)
```

**Comment by lpirola** [comment]
SQL to insert with content error:

```sql
INSERT INTO "widgets"("block_id","settings","kind","created_at","updated_at","sm_size","md_size","lg_size","mailchimp_segment_id","action_community","exported_at") VALUES (3080, '"content"=>"{\\"entityMap\\":{\\"0\\":{\\"type\\":\\"image\\",\\"mutability\\":\\"IMMUTABLE\\",\\"data\\":{\\"src\\":\\"https://s3.amazonaws.com/hub-central/uploads/1479830861_icones_ig.png\\"}}},\\"blocks\\":[{\\"key\\":\\"7kent\\",\\"text\\":\\"VAMOS TROCAR UMA IDEIA?\\",\\"type\\":\\"unstyled\\",\\"depth\\":0,\\"inlineStyleRanges\\":[{\\"offset\\":0,\\"length\\":23,\\"style\\":\\"center\\"},{\\"offset\\":0,\\"length\\":23,\\"style\\":\\"font-family: Oswald;\\"},{\\"offset\\":0,\\"length\\":23,\\"style\\":\\"font-size: 34px;\\"},{\\"offset\\":0,\\"length\\":23,\\"style\\":\\"BOLD\\"},{\\"offset\\":0,\\"length\\":23,\\"style\\":\\"color: rgba(0,0,0,1);\\"}],\\"entityRanges\\":[],\\"data\\":{}},{\\"key\\":\\"dolp4\\",\\"text\\":\\"\\",\\"type\\":\\"unstyled\\",\\"depth\\":0,\\"inlineStyleRanges\\":[],\\"entityRanges\\":[],\\"data\\":{}},{\\"key\\":\\"3kgnb\\",\\"text\\":\\"    \\",\\"type\\":\\"atomic\\",\\"depth\\":0,\\"inlineStyleRanges\\":[{\\"offset\\":0,\\"length\\":4,\\"style\\":\\"center\\"}],\\"entityRanges\\":[],\\"data\\":{}},{\\"key\\":\\"6tvcn\\",\\"text\\":\\" \\",\\"type\\":\\"atomic\\",\\"depth\\":0,\\"inlineStyleRanges\\":[],\\"entityRanges\\":[{\\"offset\\":0,\\"length\\":1,\\"key\\":0}],\\"data\\":{}},{\\"key\\":\\"8ntct\\",\\"text\\":\\"\\",\\"type\\":\\"unstyled\\",\\"depth\\":0,\\"inlineStyleRanges\\":[],\\"entityRanges\\":[],\\"data\\":{}},{\\"key\\":\\"1n6gb\\",\\"text\\":\\" contato@meurio.org.br \\\\n(21) 3079-1333\\\\n \\",\\"type\\":\\"unstyled\\",\\"depth\\":0,\\"inlineStyleRanges\\":[{\\"offset\\":0,\\"length\\":38,\\"style\\":\\"color: rgba(0,0,0,1);\\"},{\\"offset\\":0,\\"length\\":40,\\"style\\":\\"center\\"},{\\"offset\\":1,\\"length\\":21,\\"style\\":\\"BOLD\\"},{\\"offset\\":24,\\"length\\":14,\\"style\\":\\"BOLD\\"}],\\"entityRanges\\":[],\\"data\\":{}}]}"', 'content', '2016-11-22 15:47:03.860781', '2016-11-22 16:07:48.423715',12,6,8,NULL,FALSE,NULL);
```

Must change block_id reference.

**Comment by lpirola** [comment]
Again the same error with differente content:

```sql
INSERT INTO "widgets"("block_id","settings","kind","created_at","updated_at","sm_size","md_size","lg_size","mailchimp_segment_id","action_community","exported_at") VALUES (3084,'"content"=>"{\\"entityMap\\":{\\"0\\":{\\"type\\":\\"image\\",\\"mutability\\":\\"IMMUTABLE\\",\\"data\\":{\\"src\\":\\"https://s3.amazonaws.com/hub-central/uploads/1479846752_icones_fb.png\\"}},\\"1\\":{\\"type\\":\\"image\\",\\"mutability\\":\\"IMMUTABLE\\",\\"data\\":{\\"src\\":\\"https://s3.amazonaws.com/hub-central/uploads/1479846769_icones_tt.png\\"}},\\"2\\":{\\"type\\":\\"image\\",\\"mutability\\":\\"IMMUTABLE\\",\\"data\\":{\\"src\\":\\"https://s3.amazonaws.com/hub-central/uploads/1479846758_icones_ig.png\\"}}},\\"blocks\\":[{\\"key\\":\\"d5jgu\\",\\"text\\":\\"VAMOS TROCAR UMA IDEIA?\\",\\"type\\":\\"unstyled\\",\\"depth\\":0,\\"inlineStyleRanges\\":[{\\"offset\\":0,\\"length\\":23,\\"style\\":\\"left\\"},{\\"offset\\":0,\\"length\\":23,\\"style\\":\\"font-size: 30px;\\"}],\\"entityRanges\\":[],\\"data\\":{}},{\\"key\\":\\"998q3\\",\\"text\\":\\"\\",\\"type\\":\\"unstyled\\",\\"depth\\":0,\\"inlineStyleRanges\\":[],\\"entityRanges\\":[],\\"data\\":{}},{\\"key\\":\\"24rvn\\",\\"text\\":\\"                  \\",\\"type\\":\\"atomic\\",\\"depth\\":0,\\"inlineStyleRanges\\":[{\\"offset\\":0,\\"length\\":18,\\"style\\":\\"left\\"}],\\"entityRanges\\":[{\\"offset\\":17,\\"length\\":1,\\"key\\":0}],\\"data\\":{}},{\\"key\\":\\"9sut6\\",\\"text\\":\\"              \\",\\"type\\":\\"atomic\\",\\"depth\\":0,\\"inlineStyleRanges\\":[{\\"offset\\":0,\\"length\\":14,\\"style\\":\\"left\\"}],\\"entityRanges\\":[{\\"offset\\":13,\\"length\\":1,\\"key\\":1}],\\"data\\":{}},{\\"key\\":\\"2pk7k\\",\\"text\\":\\"\\",\\"type\\":\\"unstyled\\",\\"depth\\":0,\\"inlineStyleRanges\\":[],\\"entityRanges\\":[],\\"data\\":{}},{\\"key\\":\\"247u\\",\\"text\\":\\"        \\",\\"type\\":\\"unstyled\\",\\"depth\\":0,\\"inlineStyleRanges\\":[{\\"offset\\":0,\\"length\\":8,\\"style\\":\\"left\\"}],\\"entityRanges\\":[{\\"offset\\":7,\\"length\\":1,\\"key\\":2}],\\"data\\":{}}]}"','content','2016-11-22 18:55:46.387367','2016-11-22 20:49:59.125147',12,6,6,NULL,FALSE,NULL);
```

Must change block_id reference.

**Comment by lpirola** [comment]
http://www.amarildosepriscilas.meurio.org.br/ e mobilization_id 291

**Comment by igr-santos** [comment]
Fix in commit aa6c367c3663727398ab6c67d311ee69b99ee6b1

**Comment by lpirola** [comment]
Since the new boilerplate is arriving, this issue is deprecated.
